light-zk.js/src/transaction/fetchRecentTransactions.ts

```diff
@@ -225,19 +225,20 @@ async function enrichParsedTransactionEvents(
   // 3: transactionMerkleTree
   // 4: authority
   // 5: tokenProgram
-  // 6: senderSpl
-  // 7: recipientSpl
-  // 8: senderSol
-  // 9: recipientSol
-  // 10: relayerRecipientSol
-  // 11: tokenAuthority
+  // 6: tokenAuthority
+  // 7: senderSpl
+  // 8: recipientSpl
+  // 9: senderSol
+  // 10: recipientSol
+  // 11: relayerRecipientSol
```

Comment:
I think it would be best to order the following way:

6: relayerRecipientSol
7: senderSol
8: recipientSol
9: senderSpl
...

This way we put the most common accounts first and preserve the order for these:
- we will always keep the relayer recipient
- we currently have one verifier with just sol but none with just spl

(In general this is probably just a hotfix to get the ordering is the indexer right enough. We will need to add a detection in the indexer which verifier the tx is of and finetune accounts accordingly.)